### PR TITLE
Feat/proposal crud tests

### PR DESCRIPTION
**Pull request by IanFonzie**
This PR closes issue: [DMM-136]

Includes tests? [Y]
Updated docs? [N]

Proposed changes:
- Expand object mother test artifacts
- Add integration tests for swu, cwu, and twu proposals
- Expand integration test helpers

### CODE REVIEW

**Comment by IanFonzie** [comment]
@sutherlanda , can you take a quick look at this to see if there's anything out of place? I got the CI working with the new jest commands, so I'm fairly confident everything's working as expected.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,9 +16,11 @@ jobs:
         with:
           node-version: "16"
           cache: "yarn"
+      - run: |
+          echo "${{ secrets.TEST_ENV }}" > .env
```
Comment (sutherlanda) [inline .github/workflows/ci.yml:20]: Does this need to be a secret? If not, may be simpler to just hardcode it here in the workflow?

(I don't know what this is for, so feel free to ignore this comment)

**Review by sutherlanda — APPROVED** [review]
These are great improvements. I tried running the tests using the new `watch` functionality, and it works really well. Nice job.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (IanFonzie) [reply .github/workflows/ci.yml:20]: Hmm yeah, probably not now that I think about it.

**Pull request merged**